### ISSUE TITLE
trouble installing

### ISSUE THREAD

**Issue by Mullefa** [hadley/purrr#75]
Hi,

Apologies if I'm being an idiot here (wouldn't surprise me):

```R
install.packages("purrr")

Installing package into ‘/Users/guy.dawson/Library/R/3.1/library’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘purrr’ is not available (for R version 3.2.0)
```

Here's my session info:

```R
devtools::session_info()

Session info ---------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.2.0 (2015-04-16)
 system   x86_64, darwin13.4.0        
 ui       RStudio (0.99.441)          
 language (EN)                        
 collate  en_GB.UTF-8                 
 tz       Europe/London               

Packages -------------------------------------------------------------------------------------
 package    * version date       source        
 devtools   * 1.7.0   2015-01-17 CRAN (R 3.1.2)
 rstudioapi * 0.2     2014-12-31 CRAN (R 3.1.2)
```

**Comment by hadley** [comment]
It's not on CRAN yet?

**Comment by Mullefa** [comment]
At least I was half right ;)

Any idea when it will be on CRAN?

**Comment by hadley** [comment]
By the end of the year hopefully

**Comment by Mullefa** [comment]
Great - cheers Hadley.

**Comment by yuuniverse4444** [comment]
Is package purrr on CRAN now ?? 

Still get error message 
Warning in install.packages :
  package ‘purrr’ is not available (for R version 3.2.2)

**Comment by hadley** [comment]
You can easily verify that yourself: http://bfy.tw/64ov
